runtime/onert/core/include/util/logging.h

```diff
@@ -64,4 +64,10 @@ static Context &ctx = Context::get();
   if (::onert::util::logging::ctx.enabled()) \
   std::cout << "[" << __func__ << "] "
 
+#define WHENLOGENABLED(METHOD)               \
+  if (::onert::util::logging::ctx.enabled()) \
+  {                                          \
+    METHOD                                   \
+  }
+
```

Comment:
@seanshpark Perhaps enclosing with `do/while` would be safer.

```c
do { \
  if (::onert::util::logging::ctx.enabled()) { \
    ...
  } \
} while (0);

Reply:
I've followed above `VERBOSE` macro. I think using difference implementation would be better in another PR.

Reply:
Maybe 
```cxx
if (::onert::util::logging::ctx.enabled()) \
  do {
    METHOD
  } while (0);
```
?

Reply:
> Maybe
> 
> ```c++
> if (::onert::util::logging::ctx.enabled()) \
> do { METHOD } while (0);
> ```
> 
> ?

I suggested to enclose `if ...` with `do/while` because I would like to avoid cases like:

```c
if (condition)
  WHEN_LOG_ENABLED(statementA);
else
  foo();
```

It is imaginary (unlikely) example. However, I would like to ensure.

> I've followed above `VERBOSE` macro. I think using difference implementation would be better in another PR.

Okay. `VERBOSE` seems to have similar problem. It may be beyond this PR.